hugo/layouts/report.html

```diff
@@ -294,18 +314,20 @@ <h2 class="mb-3">Your Home's Top Characteristics</h2>
 
   <!-- Comp map -->
   <h2 class="mb-3">Top 5 Most Important Sales</h2>
-  <p>Tap or click on a home to see more details.</p>
+  <p>
+    <i class="bi bi-hand-index-thumb" aria-hidden="true"></i> Tap or click on a home to see more details
+  </p>
   <div
-    id="map-{{ if .is_multicard }}{{ .card.card_num }}{{ else }}main{{ end }}"
+    id="map-{{ .card.card_num }}"
     class="map-container"
   >
   </div>
 
   <!-- Comp chars -->
-  <div class="table-responsive mb-2">
+  <div class="mb-4">
     <table
-      class="table table-striped table-bordered characteristic-comparison-table"
-      id="comp-table-{{ if .is_multicard }}{{ .card.card_num }}{{ else }}main{{ end }}"
+      class="display characteristic-comparison-table"
```

Comment:
These changes remove the Bootstrap styling from the table, causing it to fall back to the default DataTables styling.